zero/meta-arducam/recipes-multimedia/libcamera-arducam/libcamera-arducam_0.0.5.bb

```diff
@@ -0,0 +1,68 @@
+SUMMARY = "Arducam libcamera framework"
+SECTION = "libs"
+
+LICENSE = "GPL-2.0-or-later & LGPL-2.1-or-later"
+
+LIC_FILES_CHKSUM = "\
+    file://LICENSES/GPL-2.0-or-later.txt;md5=fed54355545ffd980b814dab4a3b312c \
+    file://LICENSES/LGPL-2.1-or-later.txt;md5=2a4f4fd2128ea2f65047ee63fbca9f68 \
+"
+
+SRC_URI = " \
```

Comment:
Yes, I was also wondering if I should overlay with bbapend.
I will try to change to use a bbapend.

Reply:
Unfortunately, do_install fails because the libcamera version targeted by kirkstone's libcamera recipe and
the arducam libcamera version  do not match.

So, we forked the recipece from commit below and modified it